ChangeLog

```diff
@@ -6,6 +6,9 @@ Bugfix
    * Fix memory leak in mbedtls_ssl_set_hostname() when called multiple times.
      Found by projectgus and jethrogb, #836.
    * Fix usage help in ssl_server2 example. Found and fixed by Bei Lin.
+   * Parse signature algorithm extension when renegotiating. Previously,
+     renegotiated handshakes would only accept signatures using SHA-1
+     regardless of the peers preferences, or fail if SHA-1 was disabled.
```

Comment:
There's a typo `peers` -> `peer's` here - sorry, overlooked that in the initial PR.